Projects/calculator-js/script.js

```
@@ -0,0 +1,77 @@
+function updateDisplay() {
+  display.value = currentExpression;
+}
+
+function sanitizeExpression(expr) {
+  return expr.replace(/[^0-9+\-*/().%]/g, '');
+}
+
+function calculateExpression(expr) {
+  const withPercent = expr.replace(/(\d+(\.\d+)?)%/g, '($1/100)');
+  try {
+    const result = eval(withPercent);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)